dynamic_rest/content_type_field.py

```diff
@@ -0,0 +1,73 @@
+import traceback
+
+from dynamic_rest.fields import DynamicField
+from dynamic_rest.routers import DynamicRouter
+
+
+class DynamicContentTypeField(DynamicField):
```

Comment:
Nit on the name of this class: our current `DynamicRelationField` is basically a wrapper around a Django `ForeignKey`/`ManyToMany`/`RelatedObject` relationship; why not create a `DynamicGenericRelationField` that wraps `GenericForeignKey`?